pkg/kubelet/pluginmanager/operationexecutor/operation_executor_test.go

```diff
@@ -170,6 +170,10 @@ func setup() (chan interface{}, chan interface{}, OperationExecutor) {
 // This function starts by writing to ch and blocks on the quit channel
 // until it is closed by the currently running test
 func startOperationAndBlock(ch chan<- interface{}, quit <-chan interface{}) {
-	ch <- nil
-	<-quit
+	select {
+	case ch <- nil:
+		<-quit
```

Comment:
just being curious, why do you need to chain them?
/assign

Reply:
I was trying not to change the original semantic of this function: even after `ch <- nil` is executed, this function still wait for `close(quit)` to be executed

Reply:
alright
/lgtm